.github/scripts/cmake.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
```

Comment:
This is mostly https://github.com/pytorch/vision/blob/main/packaging/build_cmake.sh with all the slug taken out.